py/mpstate.h

```diff
@@ -220,6 +227,12 @@ typedef struct _mp_state_thread_t {
     mp_obj_dict_t *dict_locals;
     mp_obj_dict_t *dict_globals;
 
+    #if MICROPY_PY_SYS_PROFILING
+    mp_obj_t prof_instr_tick_callback;
```

Comment:
"prof_" should be dropped from these, not related to profile. Compare to what I had from my experiments with tracing JIT:

~~~
extern mp_code_state_t *mp_current_code_state;
~~~